self_scripts/toggle_test_mode.py

```diff
@@ -0,0 +1,45 @@
+#-*- coding:UTF-8 -*-
+import sys
+import os
+import subprocess
+
+adb_path = os.getenv('adb_path')
+serial = os.getenv('serial')
+
+get_window_animation_scale_cmd = "adb shell settings get global window_animation_scale"
+get_transition_animation_scale_cmd = "adb shell settings get global transition_animation_scale"
+get_animator_duration_scale_cmd = "adb shell settings get global animator_duration_scale"
```

Comment:
Duplicated commands.

Reply:
Saw this in https://github.com/zjn0505/adb-alfred/blob/master/self_scripts/toggle_airplane_mode.py so I wasn't sure if I should just follow the same pattern or remove them. 

I'll remove them then.